test/integration/transaction.test.js

```diff
@@ -325,12 +329,11 @@ if (current.dialect.supports.transactions) {
       this.sinon.stub(this.sequelize.queryInterface, 'commitTransaction').rejects(new Error('Oh no, an error!'));
 
       await expect(
-        (async () => {
+        (async function () {
           const t = await this.sequelize.transaction();
           t.afterCommit(hook);
           await t.commit();
-          expect(hook).to.have.been.called;
```

Comment:
Was this removed on purpose?

Reply:
Yes. the line before throws an error. The hook is checked to _not_ be called later in the same test.